test/e2e/ctx/ctx.go

```diff
@@ -127,6 +128,32 @@ func (ctx TestContext) DumpNamespaceArtifacts(namespace string) error {
 	return nil
 }
 
+func (ctx TestContext) DescribeResource(command string) error {
+	ctx.Logf("Running command %s", command)
+	stdout, stderr, err := ctx.ExecCommand(command)
+	if err != nil {
+		return fmt.Errorf("failed to run command: %s", strings.TrimSpace(stderr+err.Error()))
+	}
+	ctx.Logf("%s", strings.TrimSpace(stdout))
+	return nil
+}
+
+func (ctx TestContext) ExecCommand(command string) (string, string, error) {
+	var (
+		stdoutBuf bytes.Buffer
+		stderrBuf bytes.Buffer
+	)
+	cmd := exec.Command("bash", "-c", command)
+	cmd.Stdout = &stdoutBuf
+	cmd.Stderr = &stderrBuf
+	cmd.Env = []string{"KUBECONFIG=" + ctx.kubeconfigPath}
+
+	if err := cmd.Run(); err != nil {
+		return stdoutBuf.String(), stderrBuf.String(), err
+	}
+	return stdoutBuf.String(), stderrBuf.String(), nil
```

Comment:
```suggestion
	err := cmd.Run()
	return stdoutBuf.String(), stderrBuf.String(), err
```

Reply:
not necessarily opposed to how it is now - just noticed this simplification...

Reply:
Thanks - I had to rebase the PR anyways, so I picked up your suggestion in the latest force push.